src/Framework/EventSource/Keywords.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Build.Eventing
+{
+    public class Keywords
+    {
+        public const EventKeywords Task = (EventKeywords)1;
+        public const EventKeywords Item = (EventKeywords)2;
+        public const EventKeywords Project = (EventKeywords)3;
```

Comment:
How did you derive these keywords? How would we expect them to be used?

Reply:
We can change the keywords if you think there are more helpful options.

This is useful with IsEnabled. IsEnabled with two parameters specifies how important a particular event is (informational, error, etc.) and an EventKeywords. The first lets the user specify whether they want all the eventsources to fire or just the most critical ones (as determined arbitrarily), and the second lets them turn on/off the eventsources related to a specific goal. IsEnabled also checks the commandline argument to make sure the particular eventsource is enabled, so it actually checks three things.

Reply:
My initial reaction is "I don't know how I'd use these so do we need to put them in?"

I'd place a higher value on getting useful information out of the traces in something like the RPS case than in being able to filter specifically on the command line. That feels like an "only if we discover that some useful events are surprisingly expensive" operation.